ui-frontend/packages/catalog-ui-search/src/main/webapp/react-component/filter/filter-input/filter-location-input.js

```diff
@@ -90,7 +85,8 @@ class LocationInput extends React.Component {
     wreqr.vent.trigger('search:drawend', this.locationModel)
   }
   updateMap = () => {
-    const mode = this.locationModel.get('mode')
+    const mode = this.locationModel.get('mode') === 'keyword' ? 'poly' : 
```

Comment:
Is this formatted?

Reply:
i think so? i ran `yarn format -w` in the `ui-frontend/` directory